### PR TITLE
Homework3: Dzmitry Bachkouski

### PR DESCRIPTION
**Pull request by Dzmitry89**
Сheck my third homework please

### CODE REVIEW

File: src/Homework3.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# FizzBuzz
+"""
+FizzBuzz
+Напишите программу, которая печатает цифры от 1 до 100, но вместо чисел,
+кратных 3 пишет Fizz, вместо чисел кратный 5 пишет Buzz, а вместо чисел
+одновременно кратных и 3 и 5 - FizzBuzz
+"""
+
+for num in range(1, 101):
+    print('Fizz' * (not num % 3) + 'Buzz' * (not num % 5) or num)
```
Comment (belovmd) [inline src/Homework3.py:10]: great

File: src/Homework3.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+Дан список чисел. Посчитайте, сколько в нем пар элементов, равных друг другу.
+Считается, что любые два элемента, равные друг другу образуют одну пару,
+которую необходимо посчитать.
+Входные данные - строка из чисел, разделенная пробелами.
+Выходные данные - количество пар.
+Важно: 1 1 1 - это 3 пары, 1 1 1 1 - это 6 пар
+"""
+# формула для рассчета количества k одинаковых элементов в массиве n
+# выглядит: n!/((n-k)!*k!), в нашем условии n!/(n-2)!*2!=(n*(n-1)*(n-2)!)/
+# /((n-2)!*2)=n*(n-1)/2
```
Comment (belovmd) [inline src/Homework3.py:99]: great

File: src/Homework3.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    dct[el] = dct.get(el, 0) + 1
+print(sum(num * (num - 1) // 2 for num in list(dct.values())))
+
+"""
+Дан список. Выведите те его элементы, которые встречаются в списке только один
+раз. Элементы нужно выводить в том порядке, в котором они встречаются в списке
+"""
+
+lst = [1, 'd', 3, 'c', 4, 'd', 5, 3, 3, 4, 9, 10]
+print([el for el in lst if lst.count(el) == 1])
```
Comment (belovmd) [inline src/Homework3.py]: O(n**2)

**Review by belovmd — COMMENTED** [review]
Please address these issues

File: src/Homework3.py (hunk 3 above)
Comment (Dzmitry89) [reply src/Homework3.py]: changed to O(n)

File: src/Homework3.py (hunk 1 above)
Comment (Dzmitry89) [reply src/Homework3.py:10]: Thanks, applied the boolean property

File: src/Homework3.py (hunk 2 above)
Comment (Dzmitry89) [reply src/Homework3.py:99]: Thanks, these are the basics of computer science

**Review by belovmd — COMMENTED** [review]
9,5

**Pull request closed without merging**